src/macaron/database/rfc3339_datetime.py

```diff
@@ -14,34 +14,33 @@ class RFC3339DateTime(TypeDecorator):  # pylint: disable=W0223
     SQLAlchemy column type to serialise datetime objects for SQLite in consistent format matching in-toto.
 
     https://docs.sqlalchemy.org/en/20/core/custom_types.html#store-timezone-aware-timestamps-as-timezone-naive-utc
+    https://docs.sqlalchemy.org/en/20/dialects/sqlite.html#sqlalchemy.dialects.sqlite.DATETIME
     """
 
     # It is stored in the database as a string
     impl = String
     cache_ok = True
 
-    def process_bind_param(self, value: Optional[Any], dialect: Any) -> None | str:
-        """Process when storing.
+    def process_bind_param(self, value: None | Any, dialect: Any) -> None | str:
+        """Process when storing. If the provided ``datetime`` is a naive datetime object then UTC is added.
```

Comment:
```suggestion
        """Process when storing. If the timezone of the serialized ``datetime`` object is provided, this function preserves it. Otherwise, if the provided ``datetime`` is a naive ``datetime`` object then UTC is added.
```

Reply:
I made the change locally but this makes for a long line, see new PR https://github.com/oracle/macaron/pull/384. Waiting for that to be merged first.

Reply:
We used to address such length issues by line breaks. Thanks for the improvement.

Reply:
Line-break won’t work because:
```
src/macaron/database/rfc3339_datetime.py:25:1: D205 1 blank line required between summary line and description
        """Process when storing. If the timezone of the serialized ``datetime`` object is provided, this function
        preserves it. Otherwise, if the provided ``datetime`` is a naive ``datetime`` object then UTC is added.

        value: None | datetime.datetime
            The value being stored
        """
^
src/macaron/database/rfc3339_datetime.py:25:1: D400 First line should end with a period
        """Process when storing. If the timezone of the serialized ``datetime`` object is provided, this function
        preserves it. Otherwise, if the provided ``datetime`` is a naive ``datetime`` object then UTC is added.

        value: None | datetime.datetime
            The value being stored
        """
^
```

Reply:
I meant using a line break like this:
```
        """Process when storing datetime objects in SQLite.

        If the timezone of the serialized ``datetime`` object is provided, this function
        preserves it. Otherwise, if the provided ``datetime`` is a naive ``datetime`` object then UTC is added.

        value: None | datetime.datetime
            The value being stored
        """
```